draft-ietf-jsonpath-base.md

```diff
@@ -1512,7 +1598,7 @@ integer.
 ### `count` Function Extension {#count}
 
 Arguments:
-: 1. `OptionalNodesType`
+: 1. `NodesType`
 
 Result:
 : `ValueType` (unsigned integer)
```

Comment:
We should probably use a type which doesn't include `Nothing`. Failing that, the spec for this function should make it clear that `Nothing` is never returned.

Reply:
"(unsigned integer)" as compared with `length()`'s "(unsigned integer or `Nothing`)" makes that clear, I think.